controllers/search.js

```diff
@@ -4,6 +4,7 @@ var Parent = require('../lib/hof-standalone');
 var api = require('../api');
 var helpers = require('../lib/helpers');
 var moment = require('moment');
+var fields = require('../fields');
```

Comment:
Yeah. I agree, let's do that. I'll update.
(Been doing the ES6 thing on a file by file basis, but we already have consts further down.)